src/lib/spinel/radio_spinel.cpp

```diff
@@ -1727,6 +1727,20 @@ otError RadioSpinel::Receive(uint8_t aChannel)
     return error;
 }
 
+otError RadioSpinel::ReceiveAt(uint64_t aWhen, uint32_t aDuration, uint8_t aChannel)
```

Comment:
The argument `aStart` in [otPlatRadioReceiveAt()](https://github.com/openthread/openthread/blob/main/include/openthread/platform/radio.h#L853) is `uint32_t`. Does it necessary to change `aWhen` from `uint32_t` to `uint64_t`?

Reply:
Yes, this is the absolute time in radio clock.